sdk/synapse/Azure.Analytics.Synapse.Spark/src/Customization/SparkSessionClient.cs

```diff
@@ -36,7 +36,7 @@ public SparkSessionClient(Uri endpoint, string sparkPoolName, TokenCredential cr
         {
         }
 
-       public virtual async Task<SparkSessionOperation> StartCreateSparkSessionAsync(SparkSessionOptions sparkSessionOptions, bool? detailed = null, CancellationToken cancellationToken = default)
+        public virtual async Task<SparkSessionOperation> StartCreateSparkSessionAsync(SparkSessionOptions sparkSessionOptions, bool? detailed = null, CancellationToken cancellationToken = default)
```

Comment:
The spacing was just wrong here, aligning w\ the rest (not sure how style rules missed this).